index.js

```diff
@@ -24,7 +25,7 @@ class LatticeKeyring extends EventEmitter {
   //-------------------------------------------------------------------
   // Keyring API (per `https://github.com/MetaMask/eth-simple-keyring`)
   //-------------------------------------------------------------------
-  deserialize (opts = {}) {
+  async deserialize (opts = {}) {
```

Comment:
What happens if the requester does something like `await deserialize()`?

Reply:
That doesn't throw an error, but a linter would warn against it.

I suppose if the parent class is expecting it to be a promise, that it could make sense to leave it async.